docs/reference/aggregations/pipeline/movfn-aggregation.asciidoc

```diff
@@ -0,0 +1,575 @@
+[[search-aggregations-pipeline-movfn-aggregation]]
+=== Moving Function Aggregation
+
+Given an ordered series of data, the Moving Function aggregation will slide a window across the data and allow the user to specify a custom
+script that is executed on each window of data.  For convenience, a number of common functions are predefined such as min/max, moving averages,
+etc.
+
+This is conceptually very similar to the <<search-aggregations-pipeline-movavg-aggregation, Moving Average>> pipeline aggregation, except
+it provides more functionality and better flexibility.
```

Comment:
`more functionality and better flexibility` -> I think we can just say `more functionality` here rather than repeating?